terraform/modules/keyvault/main.tf

```diff
@@ -20,21 +20,19 @@ resource "azurerm_key_vault_access_policy" "user_accesspolicy" {
   tenant_id    = data.azurerm_client_config.current.tenant_id
   object_id    = data.azurerm_client_config.current.object_id
 
-  key_permissions = [
-    "Sign"
-  ]
+  key_permissions = ["Sign"]
 
-  certificate_permissions = [
-    "Get", "List", "Update", "Create", "Import", "Delete", "Recover", "Backup", "Restore", "ManageIssuers", "GetIssuers", "ListIssuers", "SetIssuers", "DeleteIssuers"
-  ]
+  certificate_permissions = ["Get", "List", "Update", "Create", "Import", "Delete", "Recover", "Backup", "Restore", "ManageIssuers", "GetIssuers", "ListIssuers", "SetIssuers", "DeleteIssuers"]
 }
 
 resource "null_resource" "run-api-facade" {
   triggers = {
-    key     = "${local.certs_path}.key"
+    key     = "${local.certs_path}.key.pem"
     csr     = "${local.certs_path}.csr"
     csr_der = "${local.certs_path}.csr.der"
     cert    = "${local.certs_path}-cert"
```

Comment:
Let's align on the naming here. `key`, `csr`, `csr_der` and `certpem` are self-explanatory. `cert` and `crt` are too similar. Maybe rename them to be a bit verbose? `cert` is generated from the API Facade console app. Is `crt` the same file in the DER format?

Reply:
done!